public/sass/main.sass

```diff
@@ -0,0 +1,428 @@
+// ******************************************************* TODO ****************************************************
```

Comment:
I think there some cleanup need in this file. I guess you took it from you website for "casque noir"

Reply:
Done 👍

Reply:
Well I really just use this file for importing bulma. So maybe I could just not use this file and import bulma css directly if that's possible?